modules/ptp-configuring-linuxptp-services-as-ha-bc-for-dual-nic.adoc

```diff
@@ -0,0 +1,150 @@
+// Module included in the following assemblies:
+//
+// * networking/ptp/configuring-ptp.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ptp-configuring-linuxptp-services-as-ha-bc-for-dual-nic_{context}"]
+= Configuring linuxptp as highly available boundary clock for dual NIC Intel e810 hardware
+
+You can configure `linuxptp` services (`ptp4l`, `phc2sys`) as a highly available boundary clock (T-BC) for dual Intel e810 Salem channel dual NIC hardware.
+You connect each NIC to the same upstream leader clock with separate `ptp4l` instances for each NIC feeding the downstream clocks.
+
+Create two `PtpConfig` custom resource (CR) objects for each NIC and a third `PtpConfig` CR that configures high availability between the two NICs.
+
+The third `ptpConfig` CR configures the `ptp4l` and `phc2sys` services for network clock synchronization.
+The third CR adds profiles for the `ptp4l` configurations under the `spec.profile.ptpSettings.haProfiles` key and passes the kernel socket path of those profiles to the `phc2sys` service.
+When a `ptp4l` failure occurs, the `phc2sys` service switches to the  backup `ptp4l` configuration.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+
+* Log in as a user with `cluster-admin` privileges.
+
+* Install the PTP Operator.
+
+* Configure a cluster node with Intel e810 Salem channel dual NIC.
+
+.Procedure
+
+. Create two separate `PtpConfig` CRs, one for each NIC, using the reference CR in "Configuring linuxptp services as a boundary clock" as the basis for each CR.
```

Comment:
Again We should use Dual Nic Configuration ; so I think Two Boundary Clock instances will participate in the HA setup, each with its own configuration profile with Phc2SysOpts set to empty string.